src/main/io/gps.c

```diff
@@ -1028,7 +1047,8 @@ void gpsConfigureNmea(void)
                     // disable GSV MESSAGES
                     // *** THIS COMMAND FAILS TO DISABLE GSV MESSAGES WHEN SENT ****
                     // bug?? why??
-                    serialPrint(gpsPort, "$PSRF103,03,00,00,01*27\r\n");  // disable GSV (Sat info) messages
+                    // serialPrint(gpsPort, "$PSRF103,03,00,00,01*27\r\n");  // disable GSV (Sat info) messages (with checksum enabled)
+                    serialPrint(gpsPort, "$PSRF103,03,00,00,00*26\r\n");  // disable GSV (Sat info) messages (with checksum disabled)
```

Comment:
(Same as other comments on using SiRF Tech proprietary commands.)

Reply:
Yes have been looking for recent reference manuals, the one in the opening comment is the one used. Ardupilot seems to use SiRF protocol and added it for testing.

Reply:
Each manufacturer published their own protocol docs.
The NMEA commands that correspond to the standard are (at least should be) the same in each.
But the proprietary commands are only the ones for their hardware.
The NMEA 0183 standard the GPS manufacturers use (not the latest NMEA 2000) is pretty old, from 1983, but still widely used.
I think they continue to use it since it's a text-based protocol, NMEA 200 I believe switched to a binary protocol.
So most hardware docs could be old as well, unless they updated theirs because they came out with new hardware commands.

GlobalTop Tech (MTK) from 2012: https://cdn-shop.adafruit.com/datasheets/PMTK_A11.pdf
SiRF Tech seems out of business, Sparkfun has their last doc from 2007: https://www.sparkfun.com/datasheets/GPS/NMEA%20Reference%20Manual-Rev2.1-Dec07.pdf
And of course u-blox (latest from 2023): https://content.u-blox.com/sites/default/files/u-blox-M10-SPG-5.10_InterfaceDescription_UBX-21035062.pdf